### PR TITLE
Allow chaining farts together

### PR DESCRIPTION
**Pull request by mathiasbynens**
E.g.

``` js
$('#foo').fartscroll(200).fartscroll(500);
```

…or…

``` js
$('#bar').fartscroll(20).addClass('farty-mcfarterson');
```

I’ve also moved the audio-related variables to within the IIFE block so they don’t pollute the global namespace.

### CODE REVIEW

File: fartscroll.js [hunk 1]
```diff
@@ -41,8 +42,9 @@ var oggs = [
         prefix = ogg_prefix;
     }
     var lastOffset;
-    $(this).scroll(function(event) {
-        var scrollOffset = Math.floor($(this).scrollTop() / trigger_distance);
+    var $this = this;
```
Comment (mathiasbynens) [inline fartscroll.js:45]: No need for `$(this)` — in this scenario, `this` is already a jQuery collection.

File: fartscroll.js [hunk 2]
```diff
@@ -41,8 +42,9 @@ var oggs = [
...
+    return this.scroll(function(event) {
```
Comment (mathiasbynens) [inline fartscroll.js]: By using `return`, we ensure that the plugin is chainable. You know, just like real farts.

**Comment by lliiaa** [comment]
A+++ PULL REQUEST WOULD LAUGH AGAIN

**Comment by imnotjames** [comment]
Definitely think this is important but shouldn't the moving the audio related items around be a separate commit?

File: fartscroll.js (hunk 2 above)
Comment (agillesp) [inline fartscroll.js]: <code>var $this = this;</code> $this is then not used in the following line.

Comment (mathiasbynens) [inline fartscroll.js]: Doesn’t really matter (I mostly wanted to avoid using `$(this)` here and a few lines down), but yeah — good catch. Fixed.

**Comment by inthecloud247** [comment]
+1 chaining is an very important feature.

**Comment by csinchok** [comment]
I killed the jQuery dependency with help from the code in #19. These farts must have have few dependencies as possible!

**Comment by inthecloud247** [comment]
+1 for simplified fartchitecture

**Pull request closed without merging**